chromadb/__init__.py

```diff
@@ -242,9 +266,12 @@ def Client(
 
     tenant: The tenant to use for this client. Defaults to the default tenant.
     database: The database to use for this client. Defaults to the default database.
-
     """
 
+    # https://github.com/chroma-core/chroma/issues/1573
```

Comment:
do we need to link to this issue in 4 separate places? just feels a little messy- id rather add one-liner with the rationale

Reply:
Changed these all to a comment, lmk what you think. Happy to change.